data/src/main/java/com/example/data/Data.kt

```diff
@@ -0,0 +1,49 @@
+package com.example.data
+
+import android.util.Log
+import com.example.domain.Base
+import com.example.domain.FoodType
+import com.example.domain.Item
+import com.google.gson.GsonBuilder
+import retrofit2.*
+import retrofit2.converter.gson.GsonConverterFactory
+class Data {
+    // to put data into classes
+    var repo: MutableList<FoodType> = mutableListOf()
+    //var base: Call<List<Item>> = service.listItems("base")
+    var ingredients  = listOf<String>("base.json", "crunchy.json", "dressing.json","protein.json","soft.json")
+    private fun loadData (item : String): List<Base> {
+        var data : List<Base> = emptyList()
+        var retrofit: Retrofit = Retrofit.Builder()
+            .baseUrl("https://raw.githubusercontent.com/karl-park/com.ninetynine.healthysalad/master/server/")
+            .addConverterFactory(GsonConverterFactory.create(GsonBuilder().create()))
+            .build()
+        var service: DataGetter = retrofit.create(DataGetter::class.java)
+        var call: Call<Item> = service.listItems(item)
```

Comment:
it's executed in another thread.

so, before assigning network response data to the `data` variable, this method, loadData(), will terminate right away, and it will return the "emptyList()".